### ISSUE TITLE
build-mac.sh error

### ISSUE THREAD

**Issue by owlysk** [hschneider/neutralino-build-scripts#1]
Hi,
I tried your scripts but have no luck. I downloaded your release file, extract to root of my project and run ```./build-mac.sh```  but I'll get following error
```
...
Cloning App-Scaffold ...
  Copying content:
    - Binary File
cp: cannot create regular file './dist/mac_x64/xxx.app/Contents/MacOS/main': No such file or directory
    - Resources
cp: cannot create regular file './dist/mac_x64/xxx.app/Contents/Resources/': No such file or directory
    - Icon
cp: cannot create regular file './dist/mac_x64/xxx.app/Contents/Resources/': No such file or directory
...
```
I'm using Ubuntu 22.04.3 LTS. I'm think that's because in `./dist/mac_x64/xxx.app` is directory `myapp.app`:
```
./dist/mac_x64
└── xxx.app
    └── myapp.app
        └── Contents
            ├── info.plist
            ├── MacOS
            │   └── bootstrap
            └── Resources
                └── icon.icns

```

Thanks for any help

**Comment by hschneider** [comment]
This is fixed in 1.0.5.
Thank you very much for reporting this!

-- Harald

**Comment by owlysk** [comment]
Thank you, there are no errors anymore
